src/plugins/README.md

```diff
@@ -174,3 +165,76 @@ const plugins = [ClausePlugin()];
 * `onClauseUpdated` : `Function` (Required) - Called when the text of a clause changes
 * `pasteToContract` : `Function` (Required) - Loads a template via copy/paste
 * `clauseProps`: (Required) - Props passed to the `ClauseComponent`
+
+### What it returns?
+
+```js
+{
+    name,
+    augmentSchema,
+    renderBlock,
+    isEditable,
+    onChange,
+    onPaste,
+    queries: {
+      findClauseNode,
+      isOutsideOfClause: isEditable,
+      isClauseSupported
+    }
+}
+```
+
+where augmentSchema, renderBlock, isEditable, onChange, onPaste, findClauseNode, isOutsideOfClause, and isClauseSupported are functions.
```

Comment:
Change to "where `augmentSchema`, `renderBlock`, `isEditable`, `onChange`, `onPaste`, `findClauseNode`, `isOutsideOfClause`, and `isClauseSupported` are functions."

Also provide explanation of these functions.